### PR TITLE
docs(framework): use new way

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: docs/source/zh-cn/advanced/framework.md [hunk 1]
```diff
@@ -94,37 +94,32 @@ startCluster({
...
     // 返回 framework 路径
     return path.dirname(__dirname);
   }
 }
-module.exports = YadanApplication;
+
+// 覆盖了 Egg 的 Application
+module.exports = Object.assign(egg, {
+  Application,
+});
```
Comment (atian25) [inline docs/source/zh-cn/advanced/framework.md]: 按今天在群里面的讨论，这种方式还继续么？

@dead-horse @popomore

Comment (dead-horse) [reply docs/source/zh-cn/advanced/framework.md]: 先保留吧

Comment (dead-horse) [reply docs/source/zh-cn/advanced/framework.md]: 框架开发者已经是高级玩家了

Comment (atian25) [reply docs/source/zh-cn/advanced/framework.md]: Controller 基类文档那里，你看看怎么改？

**Pull request merged**